security/security_profiles_operator/spo-release-notes.adoc

```diff
@@ -12,6 +12,18 @@ These release notes track the development of the Security Profiles Operator in {
 
 For an overview of the Security Profiles Operator, see xref:../../security/security_profiles_operator/spo-overview.adoc#[Security Profiles Operator Overview].
 
+[id="spo-release-notes-0-8-0"]
+== Security Profiles Operator 0.8.0
+
+The following advisory is available for the Security Profiles Operator 0.8.0:
+
+* link:https://access.redhat.com/errata/RHSA-2023:4689[RHBA-2023:4689 - OpenShift Security Profiles Operator bug fix update]
```

Comment:
This link is giving me a 404. Is this published yet?

Reply:
The content was not live at the time of your review, however I had an RHSA in the link, so after publication it 404'ed for me too :) 

I fixed it to the correct RHBA tag and all the content works now. This is why I always test prior to merge!